SPECS/packer/packer.spec

```diff
@@ -60,6 +60,9 @@ go test -mod=vendor
 %{_bindir}/packer
 
 %changelog
+* Wed Dec 20 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 1.8.7-1
```

Comment:
I think we might need to update the build instructions because of a warning in the release notes:
```md
Warning
The version/version.go file tagged in this release refers to 1.8.6. If you are building Packer from source
please make sure to set the version information using ldflags at build time.
```
Build Instructions:
```bash
LD_FLAGS="-X github.com/hashicorp/packer/version.Version=1.8.7 -X github.com/hashicorp/packer/version.VersionPrerelease="
go build -o "$BIN_PATH"  --ldflags=$LD_FLAGS
```

Reference: https://github.com/hashicorp/packer/releases/tag/v1.8.7